### PR TITLE
feat: Random avatar generator for profiles #154

### PR DESCRIPTION
**Pull request by DaSeeker67**
## Related Issue
Random avatar generator for profiles #154

## Description

created the function to generate random avatar from boring avatar during the signup saved the generated link into the db and use the link to show the avatar in the user profile page

## Screenshots

some authentication error

### CODE REVIEW

File: src/backend/avatarGenerator.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export const generateAvatar = (name: string, variant: string = 'marble', size: number = 120, colors: string[] = []): string => {
```
Comment (Sanchitbajaj02) [inline src/backend/avatarGenerator.ts:1]: Are you sure that once the URL is generated and saved into database, it will be static and not changed?

**Review by Sanchitbajaj02 — CHANGES_REQUESTED** [review]

**Review by Manishak798 — CHANGES_REQUESTED** [review]
Resolve Merge Conflicts.

**Comment by Sanchitbajaj02** [comment]
@DaSeeker67 please resolve the merge conflicts to get this pull request merged

**Comment by Girishbari** [comment]
@DaSeeker67 do you want any help with this...

**Review by Sanchitbajaj02 — APPROVED** [review]

**Pull request merged**